daml-foundations/daml-ghc/daml-prim-src/GHC/Types.daml

```diff
@@ -138,12 +140,13 @@ magic = magic -- deleted by the compiler
 -- | HIDE Dummy type to mark types built into DAML-LF.
 data Opaque = Opaque
 
--- Text type built into DAML-LF.
+-- | A type for text strings.
 data Text =
   -- | HIDE
   Text Opaque
 
--- Decimal type built into DAML-LF.
+-- | A type for fixed-point decimals: numbers of the form
+-- | `x / 10^10` where `x` is an integer with `|x| < 10^38`
```

Comment:
perhaps `10e10` rather than `10^10`? In programming we tend to use the former as languages like C use `^` for XOR.

Reply:
sorted